client/app/ip/ip-lb/ip-ip-lb.html

```diff
@@ -282,73 +290,72 @@ <h3 data-translate="iplb_section_title_backends"></h3>
-                                <span class="fa fa-trash"
-                                      aria-hidden="true">
-                                </span>
-                            </button>
-                        </td>
-                    </tr>
-                </tbody>
-            </table>
-            <div data-pagination-front
-                 data-current-page="currentPage2"
-                 data-nb-pages="nbPages2"
-                 data-items="selectedIplb.portsRedirectionIds"
-                 data-paginated-items="selectedIplb.portsRedirectionDetails"
-                 data-transform-item="transformItemPortsRedirection(item)"
-                 data-on-transform-item-done="onTransformItemPortsRedirectionDone(items)"
-                 data-page-placeholder="{{ 'pagination_page' | translate: { t0: currentPage, t1: nbPages } }}"
-                 data-items-per-page="itemsPerPage2"
-                 data-item-per-page-placeholder="{{ 'pagination_display' | translate }}">
+            <h3 data-translate="iplb_section_title_portsredirection"></h3>
+            <button type="button"
+                    class="btn btn-primary"
+                    data-ng-click="setAction('ip-lb/redirection/add/ip-ip-lb-redirection-add', selectedIplb)"
+                    data-ng-disabled="!iplbList.length"
+                    data-translate="iplb_portsredirection_add_title">
+            </button>
+            <div class="table-responsive mt-2">
+                <table class="table table-hover">
+                    <thead>
+                        <tr>
+                            <th scope="col"
+                                data-translate="iplb_table_portsredirection_header_srcport">
+                            </th>
+                            <th scope="col"
+                                data-translate="iplb_table_portsredirection_header_dstport">
+                            </th>
+                            <th scope="col"></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <!--loader-->
+                        <tr data-ng-if="loading.ports">
+                            <td colspan="3"
+                                class="text-center">
+                                <oui-spinner></oui-spinner>
+                            </td>
+                        </tr>
+                        <!-- no ports -->
+                        <tr data-ng-if="!selectedIplb.portsRedirectionDetails.length && !loading.ports">
+                            <td colspan="3"
+                                class="text-center"
+                                data-translate="iplb_table_portsredirection_no_ports">
+                            </td>
+                        </tr>
+                        <!--ports-->
+                        <tr data-ng-repeat="portsRedirection in selectedIplb.portsRedirectionDetails">
+                            <td scope="row"
```

Comment:
The `scope` attribute on the `td` element is obsolete. Use the `scope` attribute on a `th` element instead.

Reply:
This is done. Thanks!